addons/stock/migrations/8.0.1.1/post-migration.py

```diff
@@ -926,16 +927,85 @@ def migrate_stock_qty(cr, registry):
         UPDATE stock_move SET restrict_lot_id = {}
     '''.format(openupgrade.get_legacy_name('prodlot_id'))
     openupgrade.logged_query(cr, sql)
+    # Faster migration with required indexes
+    create_index = """
+        CREATE INDEX IF NOT EXISTS {table}_{col}_index
+        ON {table} USING BTREE({col})
+    """
+    to_index = {
+        "stock_move_operation_link": [
+            "move_id",
+            "operation_id",
+            "reserved_quant_id",
+        ],
+        "stock_pack_operation": [
+            "location_id",
+            "product_id",
+        ],
+    }
+    for table, cols in to_index.items():
+        for col in cols:
+            openupgrade.logged_query(
+                cr,
+                create_index.format(table=table, col=col),
+            )
 
     with api.Environment.manage():
-        env = api.Environment(cr, SUPERUSER_ID, {})
+        env = api.Environment(cr, SUPERUSER_ID, {'prefetch_fields': False})
+        # Force prefetching fields and records that will be used a lot
+        logger.info("Prefetching records")
+        start = datetime.now()
         moves = env['stock.move'].search(
             [('state', 'in', ['assigned', 'done'])], order="date")
-        for move in moves:
+        moves.read([
+            'linked_move_operation_ids',
+            'location_dest_id',
+            'location_id',
+            'move_orig_ids',
+            'origin_returned_move_id',
+            'partially_available',
+            'picking_id',
+            'product_id',
+            'product_qty',
+            'reserved_availability',
+            'reserved_quant_ids',
+            'restrict_lot_id',
+            'restrict_partner_id',
+            'split_from',
+            'state',
+        ])
+        locations = moves.mapped("location_id").read([
+            "usage",
+        ])
+        products = moves.mapped("product_id")
+        products.read([
+            'standard_price',
+            'type',
+            'uom_id',
+        ])
+        uoms = products.mapped("uom_id").read([
```

Comment:
Same doubt than locations

Reply:
From here I only need the count, to log it.